src/rules/mod.rs

```diff
@@ -64,6 +64,7 @@ pub mod no_sparse_arrays;
 pub mod no_this_alias;
 pub mod no_this_before_super;
 pub mod no_throw_literal;
+pub mod no_undef;
```

Comment:
Add `no_undef` to "recommended" and "all" set

Reply:
Fixed it.